packages/manager/modules/core/src/redirection/redirection.constants.js

```diff
@@ -80,6 +82,7 @@ export default {
     orders: `${managerRoot.EU}/${universes.dedicated}/${URI.orders}`,
     paymentMethod: `${managerRoot.EU}/${universes.dedicated}/${URI.paymentMethod}`,
     services: `${managerRoot.EU}/${universes.dedicated}/${URI.services}`,
+    smsOrder: `${managerRoot.EUTelecom}/${URI.smsOrder}`,
```

Comment:
Please avoid to have constant of URLS and build the URL directly in your controller where the URL is needed using the URLBuilder (cf. https://github.com/ovh/manager/blob/master/packages/manager/modules/ufrontend/README.md#url-builder )

Theses constants will be removed in https://github.com/ovh/manager/pull/4181 && https://github.com/ovh/manager/pull/4197

Reply:
This has been moved to the 2APIs